django/contrib/postgres/fields/citext.py

```diff
@@ -0,0 +1,11 @@
+from django.db.models import CharField
+
+__all__ = ['CitextField']
+
+
+class CitextField(CharField):
```

Comment:
Of course the proper handling for acronyms in CamelCase is a matter of some debate ;-) I personally think `CiTextField` is a better scheme (see https://www.approxion.com/?p=303), but unfortunately PEP 8 prefers `CITextField`.

I presume the author chose `CitextField` here on the rationale that "citext" is a single word, being the name of the relevant Postgres extension.

So there are at least three options here for which a reasonable argument can be made!

(Apologies for needlessly complexifying, I just thought this was an amusingly debatable case. I'm totally fine with `CITextField`, and prefer it over `CitextField`.)